log/04-01-rewild.md

```diff
@@ -0,0 +1,51 @@
+
+Above all, it is our fully considered aspiration for the program is to make it so that every Coinbase Wallet user can pay for their electricity bill and participate in p2p energy trading commons, that is also facilitated by the corresponding grid world / grid diffusion infrastructure being in a sustainable loop, to secure the integrity of any mechanisms and allow for "island mode" operation of their microgrids. Enable new funding models under the NounsDAO model for Microworld NFTs and discover the value of simulation _and_ establish a circular economy along simulate / execute / simulate adjustments that are compatible with system level behavior of the grid.
+
+Top priorities have converged on, by demo day:
+- 1000+ microworlds + visual rendering in a grid designer UI: https://github.com/plurigrid/microworlds
+- 1000+ deployments (local and cloud) of our Protocol, to include conversational grid agent supporting ALL languages, and a GPT4All / ALPACA (distilled from LLAMA) / or better yet, an _open_ but also commercially usable foundation model available on any node; decentralized expansion planning simulation; beginnings of open game engine as described here: https://cybercat.institute/projects/
+- an app (mobile / desktop)
+- privacy for DER / "prosumer" coalitions operating in ways that already allow for coalitions to form 
+- several hardware dev kit designs prototypes warranting additional growth injection for global expansion, for which we hope to be ready by...
+   - establishing the minimum viable base of operation of the protocol in several other global outposts, but also including here in San Francisco where the author is writing this from, and other parts of the US (NYC, Texas, LA): our devices in the most effective loop configuration already facilitating value elicitation, coalition formation, and algorithmic choice about smart grid operation
+- a source of recurring revenue from a contract at the right scale of operational capability of our team or that within reach through addition of talent - either private microgrid associated with a community, municipal, or federal government; establish a foothold in defense and contracting sector for security uses, modeling after Palantir, Galois etc.
+- a set of board candidates lined up for the next stage of growth consisting of top experts in energy sector and at least one categorical cyberneticist
```

Comment:
and someone with experience as a gov contractor/ deep knwoledge base / connections with the proper institutions, our own revolving door, our own personal Dick Cheney

Reply:
sounds good, yes, let's do so for Project E